guides/common/modules/con_transport-modes-for-remote-execution.adoc

```diff
@@ -3,41 +3,43 @@
 
 You can configure your {Project} to use two different modes of transport for remote job execution.
 
+Push-based transport::
 On {SmartProxies} in `ssh` mode, remote execution uses the SSH service to transport job details.
 This is the default transport mode.
 The SSH service must be enabled and active on the target hosts.
 The remote execution {SmartProxy} must have access to the SSH port on the target hosts.
 Unless you have a different setting, the standard SSH port is 22.
 
+Pull-based transport::
+On {SmartProxies} in `pull-mqtt` mode, remote execution uses Message Queueing Telemetry Transport (MQTT) to publish jobs it receives from {ProjectServer}.
```

Comment:
Maybe better: 
 remote execution uses Message Queueing Telemetry Transport (MQTT) to initiate the job execution it receives from {ProjectServer}.

As you described some lines below, MQTT is only used for the notification and the job is then received via HTTPS.